lib/eight_ball/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EightBall
-  VERSION = '1.0.5'
+  VERSION = '2.0.0'.freeze
```

Comment:
It's already frozen via line :1

Reply:
Rubocop... I'll remove this one though, it's annoying.